policies/alcohol.md

```diff
@@ -1,66 +1,11 @@
-    1. Any evidence or observation of an individual placing any kind of illicit foreign substance into the food or drink of their own or someone else (also known as “spiking” or placing a “roofie”), will result in the proper authorities being notified and their immediate dismissal from Techlahoma and a lifetime ban on all future affiliated gatherings.
-    1. Any perceived form of aggressive or disorderly behavior toward any attendee or organizer will be met with immediate reprimand in line with the severity of the situation, up to and including a lifetime revocation of membership and the proper authorities being contacted.
-    1. Strict adherence to the Techlahoma CoC and State/Local laws must be maintained at all times, failure to do this could result in revocation of membership or any official standing within the organization.
-    1. Providing alcohol to anyone under the age of 21 will result in immediate notification of the proper authorities and immediate revocation of all affiliation with Techlahoma and its affiliates.  
-    1. Any User Groups that willingly violate the terms or conditions of this policy, or the CoC, are subject to having their status or their Alcohol Privileges revoked until further notice.  
-
-
-The Techlahoma Alcohol policy should be used as a base for all alcohol-related activity within the organization to assure the safety and security of the organization, but most importantly its members. It is always an attendee’s responsibility to self-regulate their alcohol consumption and maintain safe behavior, but this policy shall serve as a base level of expectation from the Techlahoma Foundation and its Affiliates for any events involving alcohol. 
-
-Techlahoma is committed to offering an inclusive environment for professionals and students to meet and collaborate while understanding that for many people there is often alcohol involved in many casual settings. It is with this mentality that this policy was crafted, but it is also paramount that we possess the due diligence to create a protected and responsible environment. 
-
-We appreciate your adherence to this policy, and encourage anyone with questions to please direct them to the board in writing.  
-
-*Moving forward, all donation applications shall include a specification for donors to opt-in or out to allowing their funds to be used to procure alcohol, so that any individual or institution that otherwise abstains from its use can choose better how their particular funds are spent.   
+Alcohol will only be paid for/provided by Techlahoma at Techlahoma board organized events.
+User groups may have alcohol at events, but alcohol may not be purchased with Techlahoma funds, a sponsor can be used to pay for alcohol.
+All meetups with alcohol must be held at locations licensed to serve alcohol with a licensed bartender, such as a brewery or coffee/wine bar. 
+All rules and regulations in accordance with local, state, and federal laws must be followed.
```

Comment:
Should we add a note here about "Responsible consumption and inclusive accommodation of non-partaking community members is an expectation for any Techlahoma member or attendee. Anything deemed in poor standing of these guidelines is subject to the Techlahoma Code of Conduct and relevant consequences.